### ISSUE TITLE
something about demo

### ISSUE THREAD

**Issue by HIT-LWQ** [shurans/sscnet#28]
Dear shuran,
 I'm very interested in the SSCNet. Recently, I try to run the demo,but there are some ploblems. (I test using cpu)
firstly, Check failed: num_axes()<=4(5 vs. 4) Cannot use legacy accessors on Blobs with >4 axes. So I commented out code "CHECK_LE(num_axes(), 4)
        << "Cannot use legacy accessors on Blobs with > 4 axes.";
    CHECK_LT(index, 4);
    CHECK_GE(index, -4); " in "sscnet/caffe_code/caffe3d_suncg/include/caffe/blob.hpp"
secondly, When I run the demotest_model.py, the result of classlabel.max() and min() in the code
 "def vol2points(vol,tsdf,seg_label):
    classlabel = np.argmax(vol, axis=1) 
    colorMap = np.array([[ 22,191,206],[214, 38, 40],[ 43,160, 43],[158,216,229],[114,158,206],[204,204, 91],[255,186,119],[147,102,188],[ 30,119,181],[188,188, 33],[255,127, 12],[196,175,214],[153,153,153]])
    points = []
    rgb    = []
    for x in range(classlabel.shape[1]):
        for y in range(classlabel.shape[2]):
            for z in range(classlabel.shape[3]):
                tsdfvalue = tsdf[0][0][4*x][4*y][4*z];
                if (classlabel[0][x][y][z] > 0 and seg_label[0][0][x][y][z] <= 254 and ( tsdfvalue < 0 or tsdfvalue > 0.8)):
                    points.append(np.array([x,y,z]))
                    rgb.append(np.array(colorMap[classlabel[0][x][y][z],:]))
    points = np.vstack(points)
    rgb = np.vstack(rgb)
    return {'points':points, 'rgb':rgb}"
are all zero, it's because I using the cpu?
Thank you for your help.

**Comment by shurans** [comment]
@761687049 The current code doesn't support CPU mode, could you try to run it under GPU mode?

**Comment by HIT-LWQ** [comment]
let me try again, thanks

**Comment by LongruiDong** [comment]
> let me try again, thanks

Hi,I am facing the same issue with your first problem
![image](https://user-images.githubusercontent.com/26404032/47951773-50504780-dfa0-11e8-930e-7441cf4b9a55.png)

```
Check failed: num_axes()<=4(5 vs. 4) Cannot use legacy accessors on Blobs with >4 axes
```
**I am using gpu but without Cudnn (*Comment USE_CUDNN := 1* )**，have you solved this problem?
